### PR TITLE
Coffin "fix"

### PR DESCRIPTION
**Pull request by Pyko1**
Coffins can no longer be welded shut.
Instead you can staple them shut with stapler and 10 staples.
And you can use crowbar to pry out the staples.

New "nailed" overlay.
capture

"Fixes" https://github.com/HippieStationCode/HippieStation13/issues/1110

### CODE REVIEW

**Comment by Carbonhell** [comment]
did you copypaste the main attackby code again and remove the weldingtool thing?

**Comment by Pyko1** [comment]
Basically and then wrote the new "welding/unwelding" based on how the old worked.
Worked fine on playtest at least.

**Comment by Spacedong** [comment]
but what if you use only 9 staplers??????? ????????

File: code/game/objects/structures/crates_lockers/closets/job_closets.dm [hunk 1]
```diff
@@ -119,6 +119,89 @@
...
+		return
+	if(opened)
+		if(istype(W, /obj/item/weapon/grab))
+			if(large)
+				var/obj/item/weapon/grab/G = W
+				MouseDrop_T(G.affecting, user)	//act like they were dragged onto the closet
+				user.drop_item()
+			else
+				user << "<span class='notice'>The locker is too small to stuff [W] into!</span>"
+			return
```
Comment (ghost) [inline code/game/objects/structures/crates_lockers/closets/job_closets.dm:157]: > The locker
> keke

File: code/game/objects/structures/crates_lockers/closets/job_closets.dm [hunk 2]
```diff
@@ -119,6 +119,89 @@
...
+			playsound(loc, 'sound/items/crowbar.ogg', 50, 1)
+			if(do_after(user,80,5,1, target = src))
+				if(opened || !istype(src, /obj/structure/closet) || !user || !W || !user.loc )
+					return
+				playsound(loc, 'sound/items/crowbar.ogg', 50, 1)
+				welded = 0
+				user << "<span class='notice'>You pry off the staples keeping [src] shut.</span>"
+				update_icon()
+				user.visible_message("[user.name] has pried out the staples keeping [src] shut.", "<span class='warning'>You pry out staples keeping [src] shut.</span>")
+				new/obj/item/stack/staples(src.loc)
```
Comment (ghost) [inline code/game/objects/structures/crates_lockers/closets/job_closets.dm:193]: this could use a for loop

**Comment by Cherkir** [comment]
:+1:, muh emershuns

**Comment by Nokington** [comment]
:+1:

**Comment by danhibiki** [comment]
:+1: for loop cherkir mentioned isn't necessary at all, only makes the code more compact. Not neccessary for this PR and I give my approval.

**Pull request merged**